Source/WebCore/platform/graphics/cocoa/MediaPlaybackTargetContext.h

```diff
@@ -44,7 +44,6 @@ enum class MediaPlaybackTargetContextMockState : uint8_t {
 enum class MediaPlaybackTargetContextType : uint8_t {
     None,
     AVOutputContext,
-    SerializedAVOutputContext,
```

Comment:
It is unclear to me why we're getting rid of this and the associated logic. This should probably be explained in the commit log.